package.json

```diff
@@ -73,7 +75,8 @@
     "ntl": "^5.1.0",
     "prettier": "^2.2.1",
     "qunit": "^2.14.0",
-    "qunit-dom": "^1.6.0"
+    "qunit-dom": "^1.6.0",
+    "toastr": "^2.1.4"
```

Comment:
Do we need all three dependencies to be there?

Reply:
yes ember-toastr requires @ember/jquery and toastr